src/Orleans/Serialization/IlBasedSerializerGenerator.cs

```diff
@@ -1,217 +0,0 @@
-namespace Orleans.Serialization
```

Comment:
This file was renamed to ILBasedSerializerGenerator and if you look at the individual commits in git then you'll see that it's renamed and changed, not deleted and added under a different name. For some reason GitHub displays it differently. Sorry :(